cmd/plugin/cli/install.go

```diff
@@ -63,6 +62,8 @@ func InstallCmd() *cobra.Command {
 	cmd.Flags().String(installer.StosOperatorNSFlag, consts.NewOperatorNamespace, "namespace of storageos operator to be installed")
 	cmd.Flags().String(installer.StosClusterNSFlag, consts.NewOperatorNamespace, "namespace of storageos cluster to be installed")
 	cmd.Flags().String(installer.StorageClassFlag, "", "name of storage class to be used by etcd cluster")
+	cmd.Flags().String(installer.AdminUsernameFlag, "", "storageos secret username (string version)")
+	cmd.Flags().String(installer.AdminPasswordFlag, "", "storageos secret password (string version)")
```

Comment:
I don't think we should allow setting the password with a flag.  Maybe a flag that will prompt for an interactive password?  Or allow it with another option to prompt?

Reply:
@croomes could you expand this last part a little, I don't follow:
 "Or allow it with another option to prompt?"

Reply:
It's common for CLI tools that need a password to have a flag that will trigger an interactive prompt to enter the password using the tty.  If you were to enter it as `--password=secret` then the password will be seen in ps and command history.  e.g. (from MySQL)

```
  -p, --password[=name] 
                      Password to use when connecting to server. If password is
                      not given it's asked from the tty.
```

Reply:
Conversation offline - copying here for future reference:

> Thanks, I get the part about not wanting to enter password via the flag.
> But just so I'm clear, are we saying a better option is to keep this flag and let the user use it if they want at their own risk... and have another flag to trigger a prompt to enter the password instead?
> So if `--admin-password` is empty and (let's call it) `--admin-password-prompt`  is set, then the user gets the prompt to enter the password?
> because obviously we can't just default to the prompt every time `--admin-password`  is not set

Agreed to leave 'as is' and revisit later with a better solution